### PR TITLE
Fix cmd account validator create - no validator bug

### LINKED ISSUE #1254: lodestar-cli: "account validator create" not creating validator
**Describe the bug**

The following process is based on the instructions given here: https://chainsafe.github.io/lodestar/usage/key-management/

1.  run `yarn run cli account wallet create --name primary --passphraseFile primary.pass --rootDir .altona`
2.  run `yarn run cli account validator create --name primary --passphraseFile primary.pass --rootDir .altona`

When I run these commands, as per the expectations set in the document linked above, I would expect to see entries in the `.altona/keystore/` and `.altona/secrets/ directories, but see nothing.  Also, the validator public key is not printed to the terminal.

**Expected behavior**

I would expect to see the following happen as described in the document linked above:
```
    Derive a new BLS keypair from the wallet primary.
    Create a new directory in .lodestar/keystores containing:
    An encrypted Keystore with the validator voting keypair.
    An eth1_deposit_data.rlp file with the precomputed Eth1 deposit transaction data ready to be submitted to the deposit contract.
    Store the validator voting Keystore password in .lodestar/secrets.
    Print the validator public key to the terminal
```

### PR DESCRIPTION
**Pull request by dapplion**
Fixes https://github.com/ChainSafe/lodestar/issues/1254.

FIxes a bug where if no --count or --atMost option was specified, validatorCount became `NaN` and no validator was created while not failing

### CODE REVIEW

File: packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts [hunk 1]
```diff
@@ -79,7 +79,10 @@ export async function handler(options: IValidatorCreateOptions): Promise<void> {
   const validatorDirBuilder = new ValidatorDirBuilder(accountPaths);
   const walletManager = new WalletManager(accountPaths);
   const wallet = walletManager.openByName(name);
-  const n = count || atMost - wallet.nextaccount;
+  const n = typeof count === "number" ? count
+    : typeof atMost === "number" ? atMost - wallet.nextaccount
+      : 1;
+  if (isNaN(n)) throw new Error(`Error computing validatorCount: ${n}`);
```
Comment (wemeetagain) [inline packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: Can we throw a YargsError if neither count or atMost are specified?

Comment (3xtr4t3rr3str14l) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: @wemeetagain  what's the rationale for having those fields be required?

Comment (3xtr4t3rr3str14l) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: i know that's what caused this bug, but just wondering why it wasn't required in the first place or how this worked before without it being required (if it worked before)

Comment (wemeetagain) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: I misread this. It appears its defaulting to count=1 if neither is set.
We don't need them to be required, maybe we can be more clear in comments or cli helptext what the defaults are.
We should check if both are set tho. Error or handle that case.

It worked before if you assumed that either count or atMost were required, and set one of them.

Comment (dapplion) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: @wemeetagain there's already a yargs option configuration `conflicts` that prevents both options from being set at the same time.

I've inherited `atMost` from Lighthouse, tho it's a weird option. Would it make sense to remove `atMost` and just keep `count` defaulting it to 1?

Comment (wemeetagain) [reply packages/lodestar-cli/src/cmds/account/cmds/validator/create.ts]: :+1:

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**